nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json

```diff
@@ -0,0 +1,14 @@
+    "headers": {
+      "Content-Type": "application/json"
+    }
+  }
+}
```

Comment:
Probably nothing, but these seem special chars on the code . Maybe the compiler ignores, but maybe could cause problems !